frontend/src/app/components/admin/PopularListingCard.tsx

```diff
@@ -7,12 +7,11 @@ const PopularListingCard: React.FC<{
 }> = ({ listing }) => {
   return (
     //  add redirect to approval details page on integration
```

Comment:
Can you implement this comment as well? So clicking the card will redirect to the details page